generator/autogenlist.ts

```diff
@@ -89,6 +89,11 @@ const disabledProviders: AutoGenConfig[] = [
         namespace: 'Microsoft.Media',
         disabledForAutogen: true,
     },
+    {
+        basePath: 'authorization/resource-manager',
```

Comment:
Mind adding a comment here to explain why it's been removed?

Reply:
Comment added